keps/sig-scalability/0001-enhance-network-performance-benchmark-testcases.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Enhancing Network Performance Benchmarking Testcases
+authors:
+  - "@hanamantagoudvk"
+owning-sig: sig-scalability
+participating-sigs:
+- sig-networks
+- sig-testing
+reviewers:
+  - @mm4tt
+  - @aojea
+approvers:
+  - TBD
+editor: 
+creation-date: 2020-08-24
+last-updated: 2020-09-09
+status: implemented
+---
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-goals](#non-goals)
+  - [Challenges and Open Questions [optional]](#challenges-and-open-questions-optional)
+- [Proposal](#proposal)
```

Comment:
We should add a section on reporting, how are the results going to be stored and how can we compare if we regressed or improved?

Reply:
@aojea We have captured reporting section in HLD. Please review it.